### ISSUE TITLE
TypeError when using with TypeScript

### ISSUE THREAD

**Issue by alexieyizhe** [cristianbote/goober#80]
Hi there! Amazing work with this library.

I'm using it with Preact's TypeScript template, and it's throwing an error since the type expects children to be a property on the goober `StyledVNode`:

![image](https://user-images.githubusercontent.com/17508679/71754262-6d5a0200-2e53-11ea-88be-65a9ad1fe644.png)

When I change the definition to:
```js
export const Text = styled<{ children: any }>("span")`
  color: lightblue;
`;
```
it works fine.

The examples don't seem to have to specify this, so is this something with my code or the library?

Here are my relevant dependencies:
```
    "goober": "^1.6.1",
    "preact": "^10.0.0",
    "typescript": "^3.7.4"
```

Much appreciated!

**Comment by alexieyizhe** [comment]
@cristianbote I've opened a PR that should fix this issue along with one other.

**Comment by cristianbote** [comment]
Fixed in #85
